dbcon/joblist/distributedenginecomm.cpp

```diff
@@ -308,13 +308,24 @@ void DistributedEngineComm::Setup()
 
             writeToLog(__FILE__, __LINE__, "Could not connect to PMS" + std::to_string(connectionId) + ": " + ex.what(), LOG_TYPE_ERROR);
             cerr << "Could not connect to PMS" << std::to_string(connectionId) << ": " << ex.what() << endl;
+            
```

Comment:
We have almost identical exceptions handlers and I suggest you to put the common code into a function parametrizing the messages if needed.

Reply:
I thought about this a bit and even started writing the function, but it turns out to not save a lot. Sure I could pass in a ref to newPmCount and the ex.what(). This could standardize the log messages in one place, but I'd still have to test newPmCount ==0 for the break after return. Seems a lot of cpu for very little savings in line count.